packages/concerto-core/test/data/aliasing/parent.cto

```diff
@@ -0,0 +1,24 @@
+@test(Kid)
+namespace parent@1.0.0
+
+import child@1.0.0.{MyString as Str,Child as Kid}
```

Comment:
Perhaps it's jet lag, but I find these examples hard to parse. Let's give the identifiers more descriptive names to make the tests easier to understand.

Reply:
Done!